README.md

```diff
@@ -1,2 +1,43 @@
+
+## Project Description
+In 2012, in a [review of the methods and results reporting of more than 200 fMRI paper](https://www.ncbi.nlm.nih.gov/pubmed/22796459) the author found that "_Although many journals urge authors to describe their methods to a level of detail such that independent investigators can fully reproduce their efforts, the results described here suggest that few studies meet this criterion._"
+
+A few years ago, in order to improve replicability in f/MRI research, the Committee on Best Practices in Data Analysis and Sharing ([COBIDAS](https://www.humanbrainmapping.org/i4a/pages/index.cfm?pageid=3728)) of OHBM released a [report](https://www.biorxiv.org/content/10.1101/054262v2) to promote best practices for methods and results reporting. This was recently followed by a [similar initiative for EEG and MEG](https://osf.io/a8dhx/).
+
+Contrary to what the most optimistic people might have thought, these guidelines do not seem to have been widely adopted and anecdotal evidence ([see that twitter poll and thread](https://treeverse.app/view/Xf3jfvIZ)) suggests that, even among people who know about the report, few of them use it to write or review papers.
+
+One possible reason for this might have the unwieldy nature of the report. Anticipating on this issue, the authors of the report had included a checklist that still ended up taking almost 30 of the 70 pages of the whole document. Anyone who has used this checklist tends to agree that it is a great resource but that it is a bit cumbersome to use.
+
+### Short term goal
+
+**The short term goal of this project is to make this checklist easier to use**.
+
+One way would be to turn it into a website that people can simply click their way through.
+
+Ideally this would generate a small text file (for example a json file) that summarize what option was chosen for each item of the checklist.
+
+This machine readable file could then be used to automatically generate part of the methods section of an article.
+
+#### Extensions
+
+Filling in that checklist could still be long and tedious so a way to facilitate this further would be read as much metadata as possible from existing standards like:
+1. the [Brain imaging data structure](http://bids.neuroimaging.io/)  used for that study (for example with respect to the data acquisition and experimental design).
+2. the [NIDM results](http://nidm.nidash.org/specs/nidm-results_130.html) of any mass-univariate analysis performed for this study.
+
+#### Further developments
+
+Other off-shoot of this project could include developing toolboxes / packages / plug-ins for the main neuroimaging software (SPM, FSL, AFNI, Freesurfer) that would take some processing batch file as input (e.g matlabbatch.mat for SPM) and would output json files that could be used on the COBIDAS checklist website to automatically generate methods section (in a similar way to [what fMRIprep does](https://fmriprep.readthedocs.io/en/stable/citing.html))
```

Comment:
Do you want to include fmriprep? Feels like it would be a relevant application (I know I'm going to be scoring the fmriprep documentation to find details for COBIDAS!

Reply:
Yeah I am not sure that fMRIprep would fall in that category because it bundles together so many other softwares.
But it would make sense to have an option "data preprocessed with fMRIprep" (and eventually point to a json file because fMRIprep still has options) and it would automatically fill in the corresponding part of the checklist.

Actually that makes me think that having the same possibility for other pre-processing pipelines that exist out there might be a good idea. 

Note to self: think about possibilities to link this more generally with nipype, and the PORCUPINE and GIRAFFE pipeline designing tools that Tim van Mourik has been working on.

Reply:
I will rephrase and extend that.